yamcs-api/src/main/proto/yamcs/protobuf/filetransfer/filetransfer.proto

```diff
@@ -76,6 +77,29 @@ service FileTransferApi {
       topic: "file-transfers"
     };
   }
+
+  // Receive remote file lists
+  rpc SubscribeRemoteFileList(SubscribeTransfersRequest) returns (stream ListFilesResponse) {
+    option (yamcs.api.websocket) = {
+      topic: "remote-file-list"
+    };
+  }
+
+  // Request file list from remote
+  rpc RequestFileList(ListFilesRequest) returns (google.protobuf.Empty) {
+    option (yamcs.api.route) = {
+      post: "/api/filetransfer/{instance}/{serviceName}/requestfiles"
+      body: "*"
+    };
+  }
+
+  // Get latest file list
+  rpc GetFileList(ListFilesRequest) returns (ListFilesResponse) {
+    option (yamcs.api.route) = {
+      post: "/api/filetransfer/{instance}/{serviceName}/getfiles"
```

Comment:
This seems like it should be a get method. Suggest this:

`get: "/api/filetransfer/{instance}/{serviceName}/files"`

Then also get rid of `body: "*"`